config/roles.yml

```diff
@@ -255,3 +255,31 @@ point_in_time_full_access:
         - '*'
       allowed_actions:
         - 'manage_point_in_time'
+
+# Allows users to see security analytics detectors and others
+detectors_read_access:
+  reserved: true
+  cluster_permissions:
+    - 'cluster:admin/opensearch/securityanalytics/alerts/get'
+    - 'cluster:admin/opensearch/securityanalytics/detector/get'
+    - 'cluster:admin/opensearch/securityanalytics/detector/search'
+    - 'cluster:admin/opensearch/securityanalytics/findings/get'
+    - 'cluster:admin/opensearch/securityanalytics/mapping/get'
+    - 'cluster:admin/opensearch/securityanalytics/mapping/view/get'
+    - 'cluster:admin/opensearch/securityanalytics/rule/get'
+    - 'cluster:admin/opensearch/securityanalytics/rule/search'
+
```

Comment:
Lets also add ``security_analytics_ack_alerts`` for the security operators to be able to acknowledge alerts with cluster permission as:

```
cluster_permissions:
    - 'cluster:admin/opendistro/securityanalytics/alerts/*'
```

Reply:
done